patch/lfo/lfo.cpp

```diff
@@ -0,0 +1,140 @@
+#include "daisysp.h"
+#include "daisy_patch.h"
+#include <string>
+
+using namespace daisy;
+using namespace daisysp;
+
+#define MAX_WAVE Oscillator::WAVE_POLYBLEP_TRI
+
+DaisyPatch patch;
+
+struct lfoStruct
+{
+    Oscillator osc;
+    Parameter freqCtrl;
+    Parameter ampCtrl;
+    float amp;
+    float freq;
+    int waveform;
+    float value;
+};
+
+bool menuSelect;
+int lfoSelect;
+std::string waveNames[5];
+
+lfoStruct lfos[2];
+
+void UpdateOled();
+void UpdateControls();
+
+static void AudioCallback(float **in, float **out, size_t size)
+{
+    for (size_t i = 0; i < size; i++)
+    {
+        dsy_dac_write(DSY_DAC_CHN1, (lfos[0].osc.Process() + 1) / 2 * 4095);
+        dsy_dac_write(DSY_DAC_CHN2, (lfos[1].osc.Process() + 1) / 2 * 4095);
```

Comment:
this attenuates the output to a nominal 2.5V instead of 0V, instead of using the oscillators amplitude maybe scale it after everything else so that the LFO always starts at 0V and goes up to NV based on the amp knob.

Related to other comments: the scaling, etc. could get moved to `lfoStruct::Process()`